MetaDataset.py

```diff
@@ -79,18 +79,19 @@ class MetaDataset(CachedDataset2):
 
   def __init__(self,
                datasets,
-               data_map, data_dims,
-               seq_list_file,
+               data_map,
+               seq_list_file=None,
```

Comment:
It should stay compatible with existing configs.
So leave the old support in, do not remove the `data_dims` parameter. Make it optional instead.

There is also one reason to have it explicit:
It's basically a double check. You see more directly from the config what you expect. Like an additional `assert`.